polycommit/src/sonic_pc/mod.rs

```diff
@@ -94,6 +94,80 @@ impl<E: PairingEngine> SonicKZG10<E> {
     }
 }
 
+macro_rules! execute_in_parallel {
+    ($tasks:expr) => {{
+        #[cfg(feature = "parallel")]
+        {
+            use itertools::Itertools;
+            // At most 12 threads per task
+            const THREADS_PER_TASK: usize = 12;
```

Comment:
This is a constant chosen empirically; if y'all have a better suggestion for how to set this, please let me know.